CMakeLists.txt

```diff
@@ -153,14 +150,12 @@ set(WITH_LIBJANSSON   "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with jansson
 set(WITH_LIBEDIT      "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with editline library (default: EXTERNAL)")
 set(WITH_LIBLZO       "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with lzo library (default: EXTERNAL)")
 set(WITH_LIBGPG_ERROR "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with gpg-error library (default: EXTERNAL)")
```

Comment:
Shouldn't we also delete the library '**LIBGPG-ERROR**' package commonly used in GnuPG including **gcrypt**? Except for **gcrypt**, there is no GnuPG package in CUBRID, so it seems that it can be deleted. LIBGPG-ERROR is also under the **GPL2** license.

Reply:
Libgpg-error is actually Lesser GPL and it allows you to link against it (e.g glibc is also LGPL). I've only checked that the license is ok, I missed that it would be unused :smile:
I've removed it now.

Reply:
The review does not seem to end soon due to circumstances such as vacation. The review will be over within the next week.